river/Cursor.zig

```diff
@@ -206,6 +209,22 @@ const Mode = union(enum) {
                 } else {
                     c.wlr_seat_pointer_notify_motion(wlr_seat, time, sx, sy);
                 }
+
+                // Change focus according to config
+                var new_focus: FocusTarget = undefined;
+                const mode = config.focus_follows_cursor;
+                var refocus = false;
+                // Change only when entering a new surface in normal mode
+                if (mode == .strict) {
+                    refocus = true;
+                } else if (mode == .normal and focus_change) {
+                    refocus = true;
+                }
+                if (refocus) {
+                    self.seat.focusByCoordinates(self.wlr_cursor.x, self.wlr_cursor.y, &new_focus) catch unreachable;
```

Comment:
This is inefficient since we already have the `wlr_surface` the mouse is currently over. I think all we should do here is try `View.fromWlrSurface()` and if it succeeds call `Seat.focus()`.

Reply:
Completely removed. See 125ea72